app/assets/javascripts/graphs.coffee

```diff
@@ -0,0 +1,155 @@
+$ ->
+  if gon.raw_run isnt `undefined`
```

Comment:
Yeah that's fine.  I only did it with backticks because that's what was used with timeline.coffee and was trying to make sure stuff looked the same.